### DISCUSSION TITLE
fail friendly url -  filament error 404 without index.php, filament.js error 404 with index.php

### DISCUSSION THREAD

**Discussion by asciende** [filamentphp/filament#13089] [Help]
Initially, I create a project with Laravel 11. 
I install Livewire using "composer require livewire/livewire". 
I start this on my machine with Apache and it shows the Laravel welcome page. 
The route I access is http://myip/appfolder/public (without index.php).

Then, I install FilamentPHP using "composer require filament/filament:"^3.2" -W"
after  "php artisan filament --panels" (called admin)
Now, if I try to access http://myip/appfolder/public/admin/login, it gives me a 404 error.

If I use http://myip/appfolder/public/index.php/admin/login, it shows the page, but it doesn't load livewire.js (http://myip/appfolder/public/index.php/vendor/livewire/livewire.js).

Can someone guide me on how to configure it to have a friendly URL without index.php and use Filament?

**Comment by leandrocfe** [comment]
You should map your document_root in apache to `appfolder/public`
you can also run `php artisan serve` to access the app
